staging/src/k8s.io/apimachinery/pkg/api/operation/operation.go

```diff
@@ -42,6 +42,15 @@ type Operation struct {
 	//
 	// Unset options are disabled/false.
 	Options sets.Set[string]
```

Comment:
Speaking of optimizations (wrt CEL) we can make - if we pass an `Operation` per API call, we can change this to a slice and use linear search.  We could even inline a fixed-size array (e.g. size 8) and use that for calls with a small number of options.  We should (eventually) scrutinize anything we do per-operation, and avoid allocations.

Inline array may be premature, but set->slice seems like an obvious and easy win.

Reply:
+1.  I'll open a separate PR for this since I've got another PR stacked on this one already and it would make a mess to include the options change here due to how the stacked PR works.

Reply:
https://github.com/jpbetz/kubernetes/pull/145